lib/datadog/core/telemetry/event/app_started.rb

```diff
@@ -159,6 +159,23 @@ def configuration(settings, agent_settings)
               get_telemetry_origin(settings, 'tracing.contrib.peer_service_mapping')
             )
 
+            # OpenTelemetry configuration options (using environment variable names)
+            list.push(
+              conf_value('OTEL_EXPORTER_OTLP_ENDPOINT', settings.opentelemetry.exporter.endpoint, seq_id, get_telemetry_origin(settings, 'opentelemetry.exporter.endpoint')),
```

Comment:
is `settings.opentelemetry` always defined? And do we want to send all those configuration values if the customer is not using opentelemetry?

Reply:
Yup settings.opentelemetry.* configurations should always be defined. These configurations are initialized when the core configuration object is created: https://github.com/DataDog/dd-trace-rb/blob/a827d0fc70c7edaa63740f7b093ebee6ac19f642/lib/datadog/core/configuration/settings.rb#L1046.

If a customer is not using otel then these env vars won't be set and will be filtered out here: https://github.com/DataDog/dd-trace-rb/blob/4641d8b42b7d7d6d4ad3ab99f58a57a87459f27e/lib/datadog/core/telemetry/event/app_started.rb#L236-L237.

Reply:
great, makes sense, thanks for the explanation